src/components/HolidayRecommendationListModal.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import styled from 'styled-components';
+
+const StyledModal = styled(Modal)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 1.5rem;
+
+  .modal-body {
+    background-color: ${(props) => props.theme.color.white};
+    padding: 1rem;
+  }
+
+  .modal-top-bar {
+    text-align: right;
+    margin-bottom: -1rem;
+    span {
+      cursor: pointer;
+    }
+  }
+`;
+
+const HolidayRecommendationListModal = ({
+  className,
+  isOpen,
+  handleClose,
+  holidayRecommendations
+}) => {
+  const topFiveHolidayRecommendations = holidayRecommendations.slice(0, 5);
+
+  return (
+    <StyledModal open={isOpen} onClose={handleClose} className={className}>
+      <div className="modal-body">
+        <div className="modal-top-bar">
+          <span role="img" aria-label="close-button" onClick={handleClose}>
+            ✕
+          </span>
```

Comment:
Preview for close button
<img width="603" alt="스크린샷 2020-10-12 오후 4 35 28" src="https://user-images.githubusercontent.com/54134413/95718198-11a58a80-0ca9-11eb-9cc2-456524532b8b.png">